src/EFCore.MySql/Properties/MySqlStrings.resx

```diff
@@ -118,7 +118,7 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="IdentityBadType" xml:space="preserve">
-    <value>Identity value generation cannot be used for the property '{property}' on entity type '{entityType}' because the property type is '{propertyType}'. Identity value generation can only be used with signed integer, DateTime, and DateTimeOffset properties.</value>
+    <value>Identity value generation cannot be used for the property '{property}' on entity type '{entityType}' because the property type is '{propertyType}'. Identity value generation can only be used with integer, DateTime, and DateTimeOffset properties.</value>
```

Comment:
BTW I think the resx has been changed here (with some unrelated whitespace changes), but not the generated cs from it (the new message won't actually be applied until that is done).

Reply:
Hmm, its true that I did not regenerate the CS file (doesn't work from Rider, so I would need to load the project in VS to do so). But shouldn't the message be applied anyhow? After all, it is part of the resources, from which it should be retrieved at runtime. As far as I can see, only the description (comment) in the autogen CS file would be updated, when I let T4 run in VS in this case here (not that that is not a good idea).

Reply:
You may be right - I usually manually update both the T4 and the auto-generated, have never looked too closely at this...